pkg/ebpf/offsetguess.go

```diff
@@ -218,6 +267,26 @@ func checkAndUpdateCurrentOffset(module *elf.Module, mp *elf.Map, status *tracer
 			status.offset_ino = 0
 			status.offset_netns++
 		}
+	case guessRTT:
+		if status.rtt == C.__u32(expected.rtt) && status.rtt_var == C.__u32(expected.rttVar) {
+			status.what = guessDaddrIPv6
+			break
+		}
+		// We know that these two fields are always next to each other, 4 bytes apart:
+		// https://elixir.bootlin.com/linux/v4.6/source/include/linux/tcp.h#L232
```

Comment:
Have you confirmed this on all versions between 4.4 -> 5.2?

Reply:
(specifically just incrementing the minor version and confirming would be good)

Reply:
Yes, that's the case for >= v3.0